src/lib/battery/module.yaml

```diff
@@ -121,3 +121,61 @@ parameters:
             num_instances: *max_num_config_instances
             instance_start: 1
             default: [0, -1, -1]
+
+        BAT_LOW_THR:
+            description:
+                short: Low threshold.
+                long: |
+                    Sets the threshold when the battery will be reported as low.
+                    This has to be higher than the critical threshold.
+            type: float
+            unit: norm
+            min: 0.12
+            max: 0.5
+            decimal: 2
+            increment: 0.01
+            default: 0.15
+
+        BAT_CRIT_THR:
+            description:
+                short: Critical threshold.
+                long: |
+                    Sets the threshold when the battery will be reported as critically low.
+                    This has to be lower than the low threshold. This threshold commonly
+                    will trigger RTL.
+            type: float
+            unit: norm
+            min: 0.05
+            max: 0.25
```

Comment:
While on it: I think this max of 25% is set too low. I would set the max all all 3 thresholds to 50%.

Reply:
implemented in [434e8b2](https://github.com/PX4/PX4-Autopilot/pull/23419/commits/434e8b2d8987456645f62f0f1b2c41cba45022ee)